ops/services/okta-global/main.tf

```diff
@@ -15,25 +15,31 @@ resource "okta_auth_server_scope" "sr" {
   default          = false
 }
 
-// Add the required claims
-resource "okta_auth_server_claim" "sr_user" {
-  auth_server_id = data.okta_auth_server.default.id
-  claim_type     = "RESOURCE"
-  name           = "org"
-  value          = "appuser.pdi_org"
-  scopes = [
-  okta_auth_server_scope.sr.name]
+resource "okta_auth_server_scope" "sr_prod" {
```

Comment:
Environment specific services should probably go in the persistent directory of each env.